### DISCUSSION TITLE
JPG files stripped of their attached ICC profile

### DISCUSSION THREAD

**Discussion by Paul3367** [ImageMagick/ImageMagick#5878] [Help] [answered]
Hi everyone,

I’m not a developer but a color specialist and I’m encountering the following problem: all jpg files that we receive through image magic (CLI) are stripped of their embedded ICC profiles. This does not happen with other files (e.g. pdf etc).

I have read all available information on  but as far as I can see this is not a standard feature so there should be some specific code that does this.

Does anyone have an idea what to look for? I would appreciate any suggestions so I can try and find it

**Comment by snibgo** [comment, marked as answer]
"+profile" removes profiles. "-strip" removes all metadata including profiles.

**Reply by Paul3367** [reply]
Thank you, I will see if I can get this fixed
